api/envoy/type/grpc/v2/grpc_status.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package envoy.type.grpc.v2;
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.type.grpc.v2";
+option java_outer_classname = "GrpcStatusProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.type.grpc.v3";
+
+// [#protodoc-title: GRPC status codes]
+
+// GRPC response codes supported.
+enum Status {
+  OK = 0;
+  CANCELED = 1;
+  UNKNOWN = 2;
+  INVALID_ARGUMENT = 3;
+  DEADLINE_EXCEEDED = 4;
+  NOT_FOUND = 5;
+  ALREADY_EXISTS = 6;
+  PERMISSION_DENIED = 7;
+  RESOURCE_EXHAUSTED = 8;
+  FAILED_PRECONDITION = 9;
+  ABORTED = 10;
+  OUT_OF_RANGE = 11;
+  UNIMPLEMENTED = 12;
+  INTERNAL = 13;
+  UNAVAILABLE = 14;
+  DATA_LOSS = 15;
+  UNAUTHENTICATED = 16;
+}
+
+// GRPC status.
+message GrpcStatus {
+  // Supplies GRPC response code.
+  Status status = 1 [(validate.rules).enum = {defined_only: true not_in: 0}];
```

Comment:
Does this mean that it can't be OK?

Reply:
I had the same question/comment in the other PR.